tests/go.mod

```diff
@@ -40,7 +40,7 @@ require (
 	go.opentelemetry.io/proto/otlp v0.19.0
 	go.uber.org/zap v1.24.0
 	golang.org/x/crypto v0.10.0
-	golang.org/x/sync v0.2.0
+	golang.org/x/sync v0.3.0
```

Comment:
You should get this included in the first commit. But I won't want to insist on it in this PR.

Reply:
Yeah my preference would be to address this before merge if possible.

Reply:
It isn't a big deal, let's merge it for now.

Reply:
Noted. I will take care of this going forward.